po/sv_SE.po

```diff
@@ -0,0 +1,972 @@
+msgstr "Färg för markering"
+
+#: rc.c:109
+msgid "Use proxy"
+msgstr "Använd proxy"
+
+#: rc.c:110
+msgid "URL of HTTP proxy host"
+msgstr "URL för HTTP-proxyvärd"
+
+#: rc.c:112
+msgid "URL of HTTPS proxy host"
+msgstr "URL för HTTPS-proxyvärd"
+
+#: rc.c:115
+msgid "URL of GOPHER proxy host"
+msgstr "URL för GOPHER-proxyvärd"
+
+#: rc.c:117
+msgid "URL of FTP proxy host"
+msgstr "URL för FTP-proxyvärd"
+
+#: rc.c:118
+msgid "Domains to be accessed directly (no proxy)"
+msgstr "Domäner med direktanslutning (ingen proxy)"
+
+#: rc.c:119
+msgid "Check noproxy by network address"
+msgstr "Kontrollera noproxy via nätverksadress"
+
+#: rc.c:120
+msgid "Disable cache"
+msgstr "Inaktivera cache"
+
+#: rc.c:122
+msgid "News server"
+msgstr "News-server"
+
+#: rc.c:123
+msgid "Mode of news server"
+msgstr "Läge för news-server"
+
+#: rc.c:124
+msgid "Number of news messages"
+msgstr "Antal nya meddelanden"
+
+#: rc.c:126
+msgid "Order of name resolution"
+msgstr "Ordning på namnupplösning"
+
+#: rc.c:127
+msgid "Directory corresponding to / (document root)"
+msgstr "Katalog som motsvarar / (dokumentrot)"
+
+#: rc.c:128
+msgid "Directory corresponding to /~user"
+msgstr "Katalog som motsvarar /~användare"
+
+#: rc.c:129
+msgid "Directory corresponding to /cgi-bin"
+msgstr "Katalog som motsvarar /cgi-bin"
+
+#: rc.c:130
+msgid "Confirm when quitting with q"
+msgstr "Bekräfta vid avslut med q"
+
+#: rc.c:131
+msgid "Close tab if buffer is last when back"
+msgstr "Stäng flik om buffert är den sista vid tillbakagång"
+
+#: rc.c:133
+msgid "Enable mark operations"
+msgstr "Aktivera markeringsoperationer"
+
+#: rc.c:135
+msgid "Enable Emacs-style line editing"
+msgstr "Aktivera radredigering i Emacs-stil"
+
+#: rc.c:136
+msgid "Space key triggers file completion while editing URLs"
+msgstr "Blanksteg utlöser filkomplettering vid redigering av URL:er"
+
+#: rc.c:137
+msgid "Enable vi-like numeric prefix"
+msgstr "Aktivera vi-liknande numeriskt prefix"
+
+#: rc.c:138
+msgid "Move cursor to top line when going to label"
+msgstr "Flytta markör till översta raden vid förflyttning till etikett"
+
+#: rc.c:139
+msgid "Move cursor to top line when moving to next page"
+msgstr "Flytta markör till översta raden vid förflyttning till nästa sida"
+
+#: rc.c:140
+msgid "Fold lines of plain text file"
+msgstr "Radbryt rader för vanlig textfil"
+
+#: rc.c:141
+msgid "Show line numbers"
+msgstr "Visa radnummer"
+
+#: rc.c:142
+msgid "Show search string"
+msgstr "Visa söksträng"
+
+#: rc.c:143
+msgid "List of mime.types files"
+msgstr "Lista över mime.types-filer"
+
+#: rc.c:144
+msgid "List of mailcap files"
+msgstr "Lista över mailcap-filer"
+
+#: rc.c:145
+msgid "List of urimethodmap files"
+msgstr "Lista över urimethodmap-filer"
+
+#: rc.c:146
+msgid "Editor"
+msgstr "Redigerare"
+
+#: rc.c:147
+msgid "Mailer"
+msgstr "Epostprogram"
+
+#: rc.c:148
+msgid "How to call Mailer for mailto URLs with options"
+msgstr "Hur epostprogrammet anropas för mailto-URL:er med flaggor"
```

Comment:
Personally I would hyphenate it as "e-post", but I leave it up to you :-) There are some epost strings if you agree.

Reply:
All fixed, and especially this one since I was ambivalent about "e-post" and "epost" myself when I did the translation.